lib/utils/declarationValueIndex.js

```diff
@@ -10,12 +8,16 @@ const _ = require('lodash');
  * @returns {number}
  */
 module.exports = function (decl) {
+	// Casting this to any because 'prop' and 'value' do not exist on type 'NodeRaws'
+	/** @type {any} */
+	const raws = decl.raws;
```

Comment:
Thank you for leaving the helpful comment!
Maybe they seem to be added for CSS-in-JS via the following PR, but I don't know the details... 🤷🏼 

https://github.com/stylelint/stylelint/pull/3506/files#diff-5c428862d7b2d68d6d97041f3378f3b95c52f9c05a852285c817273ccee64a01